pandas/core/resample.py

```diff
@@ -1051,6 +1051,8 @@ def __init__(self, freq='Min', closed=None, label=None, how='mean',
 
         if isinstance(loffset, compat.string_types):
             loffset = to_offset(loffset)
+        elif isinstance(loffset, np.timedelta64):
```

Comment:
much better to do the conversion up front

Timedelta already handles these conversions (timedelta and np.timedelta)

offsets should continue to use to_offset
as those are converted to absolute days for Timedelta